mentors/python/c.json

```
@@ -43,7 +42,14 @@
      "avatar_url": null,
      "bio": "Python was my first programming language; I started learning it in school and discovered a passion for the power of programming. I strongly believe that Python is one of the best languages, especially for beginners! I currently specialize in using Python for mathematics, namely numerical analysis."
  },
- 
+ {
+     "github_username": "Collisio-Adolebitque",
+     "name": "Ray Belshaw",
+     "link_text": "GitHub Profile",
+     "link_url": "https://github.com/Collisio-Adolebitque",
+     "avatar_url": null,
+     "bio": "I've had the pleasure of working in the infrastructure world for many years and decided to pivot to the software world a few years ago, picked up Python and haven't looked back.  I love that every day is a school day in the land of Python."
+ },
```

```suggestion
  {
     "github_username": "Collisio-Adolebitque",
     "name": "Ray Belshaw",
     "link_text": "GitHub Profile",
     "link_url": "https://github.com/Collisio-Adolebitque",
     "avatar_url": null,
     "bio": "I've had the pleasure of working in the infrastructure world for many years and decided to pivot to the software world a few years ago, picked up Python and haven't looked back.  I love that every day is a school day in the land of Python."
  },
```
There are quite a few formatting mistakes in this file.  Your entry only has needs a space added before the leading `{` and trailing `}`.